test/README.md

```diff
@@ -0,0 +1,50 @@
+# How to run
```

Comment:
Please give me a crash course on how you expect we should write the test and help me ramp up.